src/components/ContentUploader/UploadState.js

```diff
@@ -47,16 +48,23 @@ const UploadState = ({ canDrop, hasItems, isOver, isTouch, view, onSelect, isFol
                     />
                 ) : (
                     <UploadStateContent
-                        fileInputLabel={<FormattedMessage {...messages.uploadEmptyFileInput} />}
+                        fileInputLabel={
+                            Browser.isIE() ? (
```

Comment:
Extract the label and message out into variables on top and use them rather than inlining the logic.
You can do 

```
label = Browser.isIE() ? ...messages.uploadEmptyFileInputCapitalized : ...messages.uploadEmptyFileInput
<FormattedMessage {...label} />
```

Also, this is a bit weird to have multiple strings just something capitalized.

Reply:
Maybe look at text transform capitalize.